firefox_puppeteer/docs/conf.py

```diff
@@ -13,7 +13,6 @@
 # serve to show the default.
 
 import os
-
```

Comment:
Oops!

Deleted accidentally while I was attempting to persuade Sphinx to generate a module index.